README.md

```diff
@@ -5,3 +5,7 @@ Project setup:
 - Server (the proxy server)
 - Clients (the web3 clients)
 - Scripts (client scripts to invoke server commands)
+
+## Documentation
+
+To create documentation for the project, run `bash doc.sh`. This builds mdbook documentation from content in manual folder in /docs. Overwrites prebuilt content in docs if any. See [manual](/manual/readmd.md) for more info on using mdbooks for documentation.
```

Comment:
*To generate the documentation for the project

I would also clarify that this will generate html pages and that the CI is the one publishing the documentation to threefoldtech.github.io/web3_proxy. Also that they should not push the generated docs to github and that they should add their documentation in manual/src/ so that CI can generate the new docs and publish.

Reply:
Yes thanks i agree